pkg/ctrl/affinity.go

```diff
@@ -125,6 +126,13 @@ func (e *Helper) BuildWorkersAffinity(workers *appsv1.StatefulSet) (workersToUpd
 					},
 				},
 			}
+
+			// TODO: remove this when EAC is ready for spread-first scheduling policy
+			// Currently EAC prefers binpack-first scheduling policy to spread-first scheduling policy. Set PreferredDuringSchedulingIgnoredDuringExecution to empty
+			// to avoid using spread-first scheduling policy
+			if e.runtimeInfo.GetRuntimeType() == common.EACRuntime {
+				workersToUpdate.Spec.Template.Spec.Affinity.PodAntiAffinity.PreferredDuringSchedulingIgnoredDuringExecution = []corev1.WeightedPodAffinityTerm{}
```

Comment:
Could you please add unit test for this change? I think we should track the change by using regression test. thanks.

Reply:
Done